package.json

```diff
@@ -54,5 +54,8 @@
     "lint-staged": "^8.1.3",
     "prettier": "^1.16.4",
     "typescript": "^3.9.5"
+  },
+  "dependencies": {
+    "react-text-mask": "^5.4.3"
```

Comment:
This should go in `web-app/package.json` not here.

Reply:
What is the difference between the 3 types of `package.json` files and how do I know where to put what?

Reply:
The ones in `/web-app` and `/api-server` are the dependencies of those projects. This one at the top is for tools we use everywhere, like prettier for example. Be cause react-text-mask is used by the web app it should be in that folder.